DOCS-2411 Azure App Service View (#11581)
DataDog/documentation: content/en/serverless/_index.md

### Azure App Service

The Datadog extension for Azure App Service provides tracing capabilities for Azure Web Apps. 

> Utilize the [Azure App Service view][3] (beta) to:

Reviewer: ```suggestion
Use the [Azure App Service view][3] (beta) to:
```

- Quickly identify apps with high latency or errors

- Track the utilization of your Web Apps, Function Apps, and App Service Plans

- Get insights into the costs of your App Service Plans by visualizing the number of active instances and seeing which are running apps that are submitting traces or logs to Datadog

> - Map which apps are running on which App Service Plans to identify apps that may be impacting costs or performance

Reviewer: ```suggestion
- Map the apps running on your App Service Plans to identify apps that may be impacting costs or performance
```
